plugins/http_extract/src/http_extract/component.py

```
@@ -0,0 +1,580 @@
+                            + timedelta(seconds=self.config.cache_ttl),
+                        }
+
+                    logger.debug(f"HTTP {response.status} from {url}")
+                    return data, response_headers
+
+            except Exception as ex:
+                logger.error(
+                    "Request failed after %s attempts: %s",
+                    self.config.retries + 1,
+                    ex,
+                )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)